src/eradius_counter.erl

```diff
@@ -24,7 +24,7 @@
 
 %% @doc initialize a counter structure
 init_counter(Key = {_ServerIP, ServerPort}) when is_integer(ServerPort) ->
-	#server_counter{key = Key, upTime = now(), resetTime = now()};
+	#server_counter{key = Key, upTime = os:timestamp(), resetTime = os:timestamp()};
```

Comment:
os:timestamp/0 is not the same a erlang:now/0 was, simply replacing one with other might not be the correct solution.

See http://www.erlang.org/doc/apps/erts/time_correction.html#id76661 and http://www.erlang.org/doc/apps/erts/time_compat.erl for better ways to handle this.